Automattic/wp-calypso
# Checkout: Also show delete button when order review step is inactive

#### Changes proposed in this Pull Request

This modifies the first step of checkout, the order review step, so that it displays the cart item delete button even when the step is not expanded.

Before:

![before-delete-always](https://user-images.githubusercontent.com/2036909/118203775-c1eb9500-b42a-11eb-8fe7-b099b73c4583.png)

After:

![after-delete-always](https://user-images.githubusercontent.com/2036909/118203778-c57f1c00-b42a-11eb-9396-118b25522f18.png)


#### Testing instructions

First: is this a good idea? With this change, users are less likely to click the "Edit" button on the first step, and therefore less likely to see the product variant picker. That was the original reason for hiding it behind the "Edit" button. I personally think the UX benefit outweighs the cost, but it's worth considering.

- Visit checkout with a product in your cart.
- Verify that you can see the trash icon adjacent to each cart item no matter which checkout step is active.
- Verify that clicking the trash icon when the first step is not active allows removing the cart item.
- Verify that there are no visual regressions and that the padding/margins on the line items look ok.
